# file: C:\Users\Asus\AppData\Roaming\Python\Python39\site-packages\exceptiongroup\_formatting.py
# hypothesis_version: 6.141.1

[750, '+', '+-', '.', '...', '<unknown>', 'HTTPError', '__main__', '__name__', '__notes__', '_format_syntax_error', 'apport_excepthook', 'builtins', 'compact', 'exception', 'name', 'note', 'obj', 's', 'urllib.error', '|']